allennlp/models/vision_text_model.py

```diff
@@ -57,17 +56,19 @@ def __init__(
         ignore_text: bool = False,
         ignore_image: bool = False,
     ) -> None:
-
         super().__init__(vocab)
 
-        self.fusion_method = fusion_method
-
-        self.embeddings = text_embeddings
-        self.image_embeddings = image_embeddings
-        self.encoder = encoder
+        from allennlp.modules.backbones import VilbertBackbone
 
-        self.t_pooler = TransformerPooler(encoder.hidden_size1, pooled_output_dim)
-        self.v_pooler = TransformerPooler(encoder.hidden_size2, pooled_output_dim)
+        self.backbone = VilbertBackbone(
+            vocab,
+            text_embeddings,
+            image_embeddings,
+            encoder,
+            pooled_output_dim,
+            fusion_method,
+            dropout,
+        )
```

Comment:
This changed so we don't need to duplicate the backbone. We still duplicate the heads, but they are tiny.